### DISCUSSION TITLE
Multiple ports ssl-proxied for single domain

### DISCUSSION THREAD

**Discussion by minicooper666** [nginx-proxy/acme-companion#1108] [Q&A]
Hello! 
Trying to acheive such a config:
service1 on port 9000 at host.domain.tld
service2 on port 9001 at host1.domain.tld

So the containers share the same certs for services.
Tried just adding the same domain and a different port into the service2 compose file, but had no luck, it throws ssl protocol error.

Could this be configured with this installation?

**Comment by prairietree** [comment]
Hello

Do you have a docker compose file that does not work that you could share, or are you just trying to understand conceptually how that would work? 

Why are you wanting to use different ports if you have different host names? If you have different host names you can have both accessed on standard 443 https from outside the proxy. It does not make sense to use acme-companion unless you are going to also be using https. In that case you can use whatever port you like on the backside of the proxy and https for all domains on the front side.

I hope that helps.

**Comment by buchdag** [comment]
@minicooper666 the ability to proxy to different services on different ports on a single container is currently not supported by nginx-proxy but is being worked on, see https://github.com/nginx-proxy/nginx-proxy/issues/1504
